prow/tide/tide.go

```diff
@@ -1798,7 +1762,20 @@ func (c *syncController) presubmitsByPull(sp *subpool) (map[int][]config.Presubm
 		log.WithField("num_possible_presubmit", len(presubmitsForPull)).Debug("Found possible preseubmits")
 
 		for _, ps := range presubmitsForPull {
-			if !ps.ContextRequired() {
+			// ContextRequired checks for `optional: ` field in prowjob, this
```

Comment:
Having an `optional` field on the ProwJob that doesn't actually tell if it is optional or not seems surprising to me